discord/ext/commands/errors.py

```diff
@@ -521,19 +521,15 @@ def __init__(self, name, original):
         super().__init__(fmt.format(name, original), name=name)
 
 class ExtensionNotFound(ExtensionError):
-    """An exception raised when an extension failed to be imported.
+    """An exception raised when an extension is not found.
 
     This inherits from :exc:`ExtensionError`
 
     Attributes
     -----------
     name: :class:`str`
         The extension that had the error.
-    original: :exc:`ImportError`
-        The original exception that was raised. You can also get this via
-        the ``__cause__`` attribute.
     """
-    def __init__(self, name, original):
-        self.original = original
```

Comment:
Since this is technically a breaking change the only way to get away with it is to explicitly set it to `None` and updating the documentation to say it's hard-coded to `None` for backwards compatibility (along with a version changed in the top level class docstring I suppose).

The documented return type was always wrong anyway, given that `Exception.__cause__` is `Optional[Exception]` it doesn't seem entirely out of line that this could also be set to explicitly `None`.